packages/web/src/scss/components/Button/index.html

```diff
@@ -1,130 +1,56 @@
-      </button>
-      <button type="button" class="Button Button--tertiary Button--medium Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-      <button type="button" class="Button Button--danger Button--medium Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-    </div>
-    <div class="docs-Box my-600">
-      <button type="button" class="Button Button--inverted Button--medium Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-    </div>
-
-    <div class="mb-400">
-      <button type="button" class="Button Button--primary Button--large Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-      <button type="button" class="Button Button--secondary Button--large Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-      <button type="button" class="Button Button--tertiary Button--large Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-      <button type="button" class="Button Button--danger Button--large Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24"  aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-    </div>
-    <div class="docs-Box my-600">
-      <button type="button" class="Button Button--inverted Button--large Button--square">
-        <span class="accessibility-hidden">Menu</span>
-        <svg class="Icon" width="24" height="24" aria-hidden="true">
-          <use xlink:href="/icons/svg/sprite.svg#hamburger" />
-        </svg>
-      </button>
-    </div>
-
-  </section>
+{{#> layout/plain }} {{setVar "colors" "primary" "secondary" "tertiary" "inverted" "success" "informative" "warning"
+"danger" }} {{setVar "sizes" "small" "medium" "large" }}
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+
+  <div class="Grid Grid--tablet--cols-3">
```

Comment:
Needs a fix to work on mobile screens too.

```suggestion
  <div class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
```